Makefile

```diff
@@ -25,3 +25,12 @@ build: amppkg
 .PHONY: amppkg
 amppkg:
 	go build $(GOOPTS) -ldflags "$(VERSION_LDFLAGS)" -o amppkg ./cmd/amppkg/...
+
+.PHONY: update-go-deps
+update-go-deps:
+	@for m in $$(go list -mod=readonly -m -f '{{ if and (not .Indirect) (not .Main)}}{{.Path}}{{end}}' all); do \
```

Comment:
What's the difference between this for loop and `go get -u` without a package arg? From `go help module-get`:

> In particular, 'go get -u' and 'go get -u=patch' update all the dependencies of that package.

(If the answer is non-obvious, then please add a comment in the Makefile.)

Reply:
This is meant to skip the `// indirect` modules. Since they typically require extra care in updating, or are automatically updated when you get other packages.

Reply:
Ah OK. And I guess in this particular case you reverted the updates it made to WICG/webpackage?